### ISSUE TITLE
6 MB fallback fails for IIIF IDs that have a % in them (encoded /)

### ISSUE THREAD

**Issue by tpendragon** [samvera-labs/serverless-iiif#52]
We had to fix this by installing a viewer-request function (https://github.com/pulibrary/serverless-iiif/blob/main/src_edge/index.js), the comment in there describes the issue best I think:

```js
  // The lambda will write to an S3 bucket and fall back to it if the image is
  // too big. Unfortunately the object URL for S3 replaces % with %25, so
  // fallback doesn't work. So rewrite % to %25, and then the lambda will use
  // x-original-uri, while S3 will have the right request URI.
```

**Comment by mbklein** [comment]
@tpendragon Can you provide steps to reproduce this issue? Even though you have a workaround, I'd like to fix it in core, but I've forgotten exactly what the issue is.

**Comment by tpendragon** [comment]
I think you just have to put a file in an s3 bucket with a folder and then try to render it

**Comment by tpendragon** [comment]
A large file that has to fall back to cache

**Comment by mbklein** [comment]
I believe this was fixed by samvera-labs/node-iiif#23 (via 84da4d6). I'm closing this issue but @tpendragon if you deploy the latest version and find that this is still an issue, feel free to reopen.
